RevenueCatUI/Views/SafariView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SafariView.swift
+//
+//  Created by Antonio Rico Diez on 2024-10-18.
+
+#if os(iOS)
+
+import SafariServices
+import SwiftUI
+
+struct SafariView: UIViewControllerRepresentable {
```

Comment:
Lmk what you think of this. Seems to work fine and seemed like an easy solution :)

Reply:
Ohh I had missed that one! Thanks for the heads up! I can try to consolidate yeah